### DISCUSSION TITLE
Simulating infrared spectrum without the use of wannier centroids

### DISCUSSION THREAD

**Discussion by luukasnik** [deepmodeling/deepmd-kit#2329] [General]
Hi,

I'm trying to simulate infrared spectrum from a lammps md simulation using deepmd. From my AIMD program i get the total dipole moment for my molecule. If i include the global dipole.npy file and put pref to 1.0 and no atomic_dipole.npy, does this take into account all the atom positions in the molecule and train for that dipole moment? If that is the case does the rcut radius have any difference or should it be big as big as the simulation box? 

Thank you

**Comment by njzjz** [comment]
The total dipole is the sum of the atomic dipole, instead of being directly calculated.

**Reply by luukasnik** [reply]
So it is not possible to train a model to directly give total dipole?

**Reply by njzjz** [reply]
The total dipole is trained similarly to the atomic dipole.

**Reply by luukasnik** [reply]
Does this mean that the total dipole moment must be assigned to an atom and trained from its perspective? Can rcut be set to be as big as the molecule so that all the atoms are taken into account?

**Reply by njzjz** [reply]
The total dipole is automatically decomposed into the atomic dipole, like how we train a potential energy model. Here you confuse the model and the loss function. No matter the loss function, the model calculates the atomic dipole and sum them to obtain the total dipole. The loss function can be the error of either the atomic dipole or the global dipole.

**Reply by luukasnik** [reply]
Okay, I see how the loss function works now. 

The output of my AIMD does not give me any atomic dipoles though. The only dipole is the total for the molecule. This is how the ouput looks like:


                           electrostatic moments
 ==============================================================================

 reference point for electrostatic moments:    0.00000   0.00000   0.00000

 
              nuc           elec       ->  total
 ------------------------------------------------------------------------------
                          charge      
 ------------------------------------------------------------------------------
          26.000000     -26.000000       0.000000
 
 ------------------------------------------------------------------------------
                       dipole moment  
 ------------------------------------------------------------------------------
   x      -2.618744       2.697252       0.078508
   y       1.103416      -0.483061       0.620355
   z       0.000000      -0.000038      -0.000038
 
   | dipole moment | =     0.6253 a.u. =     1.5894 debye 

Can I in some way trick the model into learning this dipole moment (or the x y z components) to a virtual stationary atom on position 0, 0, 0 so that the local frame for that point (rcut = whole molecule) determines the entire dipole moment? I am only working with one molecule instead of a system.

**Comment by luukasnik** [comment]
Hi,

Is it possible to assign the sel_type in dipole fitting to only one atom instead of all atoms of same type?

Thanks
